.github/workflows/ci.yml

```
@@ -48,4 +47,4 @@ jobs:
 
       - run: npm ci
       - run: npm run build
-      - run: npx jest
+      - run: npm run test --ignore-script
```

same comment as [here](https://github.com/readmeio/oas-to-har/pull/223#discussion_r1292844449), but you said `ignore-script` instead of `ignore scripts` here 👀

```suggestion
      - run: npm run test --ignore-scripts
```